src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala

```diff
@@ -58,18 +58,18 @@ class ConnectionActor(
 
     case Event(_: AmqpShutdownSignal, _) => stay()
 
-    case Event(msg @ ProvideChannel, _) =>
-      log.debug("{} can't create channel for {} in disconnected state", header(Disconnected, msg), sender())
+    case Event(ProvideChannel, _) =>
+      log.debug("{} can't create channel for {} in disconnected state", header(Disconnected, ProvideChannel), sender())
       stay()
   }
   when(Connected) {
-    case Event(msg @ ProvideChannel, Connected(connection)) =>
+    case Event(ProvideChannel, Connected(connection)) =>
       safe(connection.createChannel()) match {
         case Some(channel) =>
-          log.debug("{} channel acquired", header(Connected, msg))
```

Comment:
I agree that it looks silly and would expect it to work fine. Still if we can remove easily avoidable compiler warnings that would help against warning blindness in future for more important cases.